docs/02-app/01-building-your-application/04-caching/index.mdx

```diff
@@ -0,0 +1,468 @@
+}
+
+// This function is called twice, but the result is only fetched once
+const comments = await getComments() // cache MISS
+
+// The second call could be anywhere in your application
+const comments = await getComments() // cache HIT
+```
+
+For cases where `fetch` is not suitable (e.g. database clients, CMS clients, or GraphQL), you can use React's `cache` function to manually memoize functions.
+
+> **Memoization vs. Deduplication**:
+>
+> You may see the terms **memoization** and **deduplication** being used interchangeably. Memoization is a general term that applies storing the return values of **any function** in-memory, whereas request deduplication applies specifically to **data requests**.
+
+#### Duration
+
+The cache lasts the lifetime of a server request or static rendering until rendering the React tree completes.
+
+#### Revalidating
+
+React's `cache` function is used specifically for memoizing the return values of functions. Since the memoization is not shared across requests and only applies during the rendering of the React tree, there is no need for revalidation.
+
+#### Opting out
+
+To opt out of memoization in `fetch` requests, you can pass an `AbortController` `signal` to the request.
+
+```js
+const { signal } = new AbortController()
+fetch(url, { signal })
+```
+
+Since you manually opt into using the `cache` function, there's no way to opt out.
+
+### Data Cache
+
+Next.js has a built-in Data Cache that stores the result of data requests across user requests and deployments.
+
+By default, data requests that use `fetch` are automatically cached. This is possible because Next.js extends the native `fetch` API to allow each request on the server to set its own persistent caching semantics.
+
+You can use the [`cache`](#fetchoptionscache) and [`next.revalidate`](#fetchoptionsnextrevalidate) options of `fetch` to configure the caching behavior. In the browser, `cache` indicates how a fetch request will interact with the browser's HTTP cache, and with Next.js, the `cache` option also indicates how a server-side fetch request will interact with the framework's Data Cache.
+
+When fetching data on the server, a `fetch` request first checks the Data Cache for a cached response. If a cached response is found, it is returned immediately. If not, the request is made to the data source, and the response is cached.
+
+> **Static vs. Dynamic Data Fetching**:
+>
+> You may see the terms **Static Data Fetching** and **Dynamic Data Fetching** being used. Static Data Fetching refers to cached data requests (default), whereas Dynamic Data Fetching refers to data that is fetched on every request, at runtime.
+
+#### Duration
+
+The Data Cache is persistent across user requests and deployments unless you revalidate or opt-out.
+
+#### Revalidating
+
+Revalidation is the process of purging the Data Cache and re-fetching the latest data. This is useful when your data changes and you want to ensure you show the latest information.
```

Comment:
> and re-fetching the latest data

In case of the time-based revalidate then yes the latest data is re-fetched. In case of on-demand revalidation the cache is only purged, it does not immediately fetch a new result, because these are individual fetches.

Reply:
We could remove this paragraph, since it's duplicated in the data fetching page. Then, link to it from here. I'll update the description on the data fetching page.